wpilibNewCommands/src/test/native/cpp/frc2/command/CommandDecoratorTest.cpp

```diff
@@ -383,6 +383,45 @@ TEST_F(CommandDecoratorTest, RaceWithOrder) {
   EXPECT_TRUE(firstWasPolled);
 }
 
+TEST_F(CommandDecoratorTest, Repeatedly) {
+  CommandScheduler scheduler = GetScheduler();
+
+  int counter = 0;
+
+  auto command = InstantCommand([&counter] { counter++; }, {}).Repeatedly();
+
+  scheduler.Schedule(command);
+
+  for (int i = 1; i <= 50; i++) {
+    scheduler.Run();
+    EXPECT_EQ(i, counter);
+  }
+
+  EXPECT_TRUE(scheduler.IsScheduled(command));
+}
+
+TEST_F(CommandDecoratorTest, RepeatedlyCount) {
+  CommandScheduler scheduler = GetScheduler();
+
+  int counter = 0;
+
+  auto command = InstantCommand([&counter] { counter++; }, {}).Repeatedly(3);
+
+  scheduler.Schedule(command);
+  EXPECT_TRUE(scheduler.IsScheduled(command));
```

Comment:
```suggestion
  EXPECT_EQ(1, counter);
```
This matches Java and is more useful than the current check.
There's also a few more small changes- Make sure to read the whole comment.

Reply:
added with 54aa6c9